tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html

```diff
@@ -15,6 +15,35 @@
 limitations under the License.
 -->
 
+<custom-modal #contextMenu>
+  <div class="context-menu">
+    <button
+      *ngIf="canContextMenuRemoveColumn()"
+      mat-button
+      (click)="contextMenuRemoveColumn()"
+    >
+      Disable
```

Comment:
This is the part I am most uncertain of.
* Should this be it's own component?
* Should this have keyboard support?
* Should this be "Remove" instead? Do we want to add an icon?

Reply:
I changed it to remove and moved it into the data table, I think that is a good middle ground